miden/tests/integration/operations/decorators/advice.rs

```diff
@@ -117,3 +117,18 @@ fn advice_inject_u64div_conditional_execution() {
     let test = build_test!(source, &[8, 0, 4, 0, 1, 0]);
     test.expect_stack(&[0, 0, 0, 0, 0, 4, 0, 8]);
 }
+
+#[test]
+fn advice_inject_mem() {
+    let source = "begin
+    #! write to memory and copy from memory to advice map
```

Comment:
Could we use regular comments here (i.e., `#`) instead of doc comments (i.e., `#!`).

Reply:
Ah sorry I thought the syntax had been changed for all comments. Fixed.